cmd/internal/learn/flags.go

```diff
@@ -65,17 +65,23 @@ func registerRequiredFlags() {
 }
 
 func registerOptionalFlags() {
+	Cmd.Flags().StringVar(
+		&serviceFlag,
+		"project",
+		"",
+		"Your Akita project. Exactly one of --out, --cluster, or --project must be specified.")
```

Comment:
The old description of `--service` suggests that both `--out` and `--service` were required if `--out` didn't specify an AkitaURI. This hasn't changed, has it?